plugins/woocommerce/src/Blocks/BlockTypes/AddToCartForm.php

```diff
@@ -162,6 +162,9 @@ protected function render( $attributes, $content, $block ) {
 
 			return '';
 		}
+
+		wp_enqueue_script_module( $this->get_full_block_name() );
```

Comment:
This is required by all blocks whose frontend script is generated as a script module?